databases/week3/databases_week3_mealsharing_DML.sql

```diff
@@ -0,0 +1,234 @@
+-- Quries to write on Reservation table
+  -- Get all reservations
+select
+  meal.title as Food,
+  reservation.number_of_guests as 'Total Guests',
+  meal.when as 'Date of Event'
+from
+  reservation
+  JOIN meal on reservation.meal_id = meal.id;
+-- Add a new reservation
+INSERT INTO
+  reservation(
+    `number_of_guests`,
+    `created_date`,
+    `contact_phonenumber`,
+    `contact_name`,
+    `contact_email`,
+    `meal_id`
+  )
+VALUES(
+    85,
+    '2022-02-16',
+    '75757575',
+    'marta',
+    'marta@gmail.com',
+    4
+  );
+-- Get a reservation with any id, fx 1
+select
+  *
+from
+  reservation
+where
+  id = 2;
+SELECT
+  -- DISTINCT(meal.title)
+  *
+FROM
+  meal
+  JOIN reservation on meal.id = reservation.meal_id
+WHERE
+  meal_id = 2;
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+  reservation
+SET
+  contact_phonenumber = '27859433'
+WHERE
+  meal_id = 1;
+-- Delete a reservation with any id, fx 1
+DELETE FROM
+  reservation
+WHERE
+  meal_id = 1;
+-- Quries to write on Review table
+SELECT
+  title
+from
+  review;
+-- Add a new review
+INSERT INTO
+  review(
+    `title`,
+    `description`,
+    `stars`,
+    `created_date`,
+    `meal_id`
+  )
+VALUES(
+    'best food',
+    'i ate best food ever i found in my life. thanks to the main chef there. salute',
+    5,
+    '2022-02-26',
+    3
+  );
+-- Get a review with any id, fx 1
+SELECT
+  *
+from
+  review
+where
+  id = 1;
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+  review
+SET
+  stars = 5
+WHERE
+  meal_id = 2;
+-- Delete a review with any id, fx 1
+DELETE FROM
+  review
+WHERE
+  id = 1;
+-- Additional queries on functionnality
+  -- Get meals that has a price smaller than a specific price fx 90
+SELECT
+  meal.title,
+  meal.price
+from
+  meal
+WHERE
+  price = 90;
+-- Get meals that still has available reservations
+SELECT
+  meal.title,
+  max_reservations,
+  number_of_guests,(max_reservations - number_of_guests) as 'Available Reservations'
+from
+  meal
+  JOIN reservation on meal.id = reservation.meal_id
+WHERE
+  reservation.number_of_guests < meal.max_reservations;
+-- Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+INSERT INTO
+  meal(
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+VALUES(
+    'Rød grød med fløde',
+    'danish special',
+    'china town lyngby',
+    '2022-02-16 09:09:09',
+    100,
+    50,
+    '2022-02-15'
+  );
+SELECT
+  meal.title
+from
+  meal
+WHERE
+  title LIKE 'Rød grød med%';
+-- Get meals that has been created between two dates
+SELECT
+  meal.title,
+  meal.created_date
+FROM
+  meal
+WHERE
+  created_date > '2022-02-14' & & created_date < '2022-02-16';
+-- Get only specific number of meals fx return only 5 meals
+SELECT
+  meal.title
+from
+  meal
+where
```

Comment:
We don't say anything about the price

Reply:
i removed this 
where
  price < 95

and stills it works !! yes understood . thank you

Reply:
also deleted 

ORDER BY
  meal.title ASC

still it works. thank you

Reply:
not crashing doesn't mean it does what it should be doing ;)